packages/styled-system/tests/css-var-token-map.test.ts

```diff
@@ -0,0 +1,138 @@
+import { toCSSVar } from "../src"
+
+describe("Styled System: CSS variable token map", () => {
+  it("should create css vars with condition selectors", () => {
+    const theme = {
+      fonts: {
+        en: "Inter",
+        ar: "Tajawal",
+      },
+      colors: {
+        gray: {
+          100: "lightgray",
+          800: "darkgray",
+        },
+        red: {
+          500: "crimson",
+        },
+      },
+      tokensMap: {
+        colors: {
+          error: { DEFAULT: "red.500", _dark: "red.100" },
+          errorText: { DEFAULT: "white", _dark: "black" },
+          layerBg: "green.500",
+        },
+        fonts: {
+          body: { _ltr: "en", _rtl: "ar" },
+        },
+      },
+      config: {
+        cssVarPrefix: "chakra",
+      },
+    }
+
+    const result = toCSSVar(theme)
+
+    expect(result.__cssVars).toMatchInlineSnapshot(`
+      Object {
+        "--chakra-colors-error": "var(--chakra-colors-error-DEFAULT, var(--chakra-empty,/*!*/ /*!*/))",
+        "--chakra-colors-error-DEFAULT": "var(--chakra-colors-red-500)",
+        "--chakra-colors-errorText": "var(--chakra-colors-errorText-DEFAULT, var(--chakra-empty,/*!*/ /*!*/))",
```

Comment:
This looks incorrect. The expected value should be

```suggestion
        "--chakra-colors-errorText": "var(--chakra-colors-white)",
```

Let's also remove all the "empty" fallback values since they won't have any effect.